red/locales/de-DE/s7.json

```diff
@@ -0,0 +1,118 @@
+{
+    "s7": {
+        "in": {
+            "label": {
+                "name": "s7 in",
+                "endpoint": "SPS",
+                "mode": "Modus",
+                "variable": "Variable",
+                "diff": "Sende nur bei Wertänderung (diff)",
+                "variable-select": "Variable auswählen",
+                "variable-novar": "Keine Variablen"
+            },
+            "mode": {
+                "single": "Einzelne Variable",
+                "all-split": "Alle Variablen, eine pro Nachricht",
+                "all": "Alle Variablen"
+            }
+        },
+        "out": {
+            "label": {
+                "name": "s7 out",
+                "endpoint": "SPS",
+                "variable": "Variable",
+                "variable-select": "Variable auswählen"
+            },
+            "disclaimer": "Vorsicht beim Schreiben auf produktiven Steuerungen!"
+        },
+        "control": {
+            "label": {
+                "name": "s7 Steuerung",
+                "function": "Funktion",
+                "endpoint": "SPS"
+            },
+            "function": {
+                "cycletime": "Zykluszeit",
+                "trigger": "Lesen auslösen"
+            }
+        },
+        "endpoint": {
+            "label": {
+                "address": "IP Addresse",
+                "port": "Port",
+                "connmode": "Modus",
+                "rack": "Rack",
+                "slot": "Slot",
+                "tsap-local": "Local TSAP",
+                "tsap-remote": "Remote TSAP",
+                "cycletime": "Zykluszeit",
+                "timeout": "Timeout",
+                "verbose": "Debug",
+                "tabs": {
+                    "connection": "Verbindung",
+                    "variables": "Variablen"
+                },
+                "variables": {
+                    "list": "Variablen Listen",
+                    "add": "Hinzufügen",
+                    "clean": "Alle Entfernen",
+                    "del": "Entfernen",
+                    "name": "Name",
+                    "addr": "Addresse",
```

Comment:
As far as my German goes... shouldn't this be "Adresse"?

Reply:
Yes, you are right.

I did a third read and a rewording.